packages/core/src/odata-v4/entity.spec.ts

```diff
@@ -55,8 +55,8 @@ describe('entity v4', () => {
       })
       .build();
   });
-  it('getCurrentMapKeys', () => {
-    const currentState = entity.getCurrentMapKeys();
+  it('asObject', () => {
```

Comment:
in general, `asObject` is not a v4 specific function. Should we keep it in the common or v2 folder?

Reply:
Yes, common makes more sense. Unfortunately there are so many untidy things around this, that I wasn't sure what to touch and what not... But I can move this test.

Reply:
Now, I remembered, why I didn't move it. There are clashes with some of the other tests and I want this to be done thoughtfully. I will create a ticket for this.